examples/version-migration/same-container/package.json

```diff
@@ -11,13 +11,21 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			}
+		}
+	},
 	"main": "lib/index.js",
-	"module": "lib/index.js",
 	"types": "lib/index.d.ts",
```

Comment:
I was going to suggest that library/app hybrids should really keep their test tsconfig separate from product(library) code. But this folder is app only. The index files referenced here do not exist.
I wonder if attw catches this issue.

Reply:
Good catch, thanks.

The reason we're not adding 'attw' for ESM-only packages is that it will only pass if node10, node16/CJS, node16/ESM, and bundler all resolve.  If your package is omits any of those combinations, attw fails.

It's probably trivial to add some config to attw that modifies the test matrix, but for ESM-only or CJS-only packages, building tests usually catches mistakes.

PS - I'm also not a fan of comingled product & test builds.